source/authentication/oidc.rst

```diff
@@ -47,6 +47,7 @@ The following is an example :program:`ood-portal-generator` configuration file:
      OIDCPassRefreshToken: "On"
      OIDCPassClaimsAs: "environment"
      OIDCStripCookies: "mod_auth_openidc_session mod_auth_openidc_session_chunks mod_auth_openidc_session_0 mod_auth_openidc_session_1"
+     OIDCResponseType: "code"
```

Comment:
Good question Jeff. Our site actually needed to set the value to `"'code id_token'"` (note the double quoting, httpd needs it to be quoted after the ood-portal-generator is run). I was debating whether to put in our example, or use the default myself before submitting the PR, but ultimately decided to use the default, since it's probably a default for a reason? I.e. admins perhaps need to modify the example configuration less, if the values are "as default as possible" so to speak.

If treydock then says that all sites don't require it, it's up for debate whether this should be added as an optional note or something instead? At least on our site, we saw errors until we explicitly specified an `OIDCResponseType`. Luckily then error message was very clear, so we found the reason instantly.